src/components/Recorder.js

```diff
@@ -1,16 +1,27 @@
 import MicRecorder from 'mic-recorder-to-mp3';
 import {nanoid} from 'nanoid';
+import {useRouter} from 'next/router';
 import {useEffect, useState, useRef} from 'react';
+import EdiText from 'react-editext';
 import {FaMicrophone} from 'react-icons/fa';
 import {FaMicrophoneAltSlash} from 'react-icons/fa';
 
 export default function Recorder() {
+	const router = useRouter();
+
+	function handleSubmit() {
+		router.push('/start');
+	}
+
 	const recorder = useRef(null);
 
 	const [isRecording, setIsRecording] = useState(null);
 
 	const handleAddRecording = (blobUrl, audiofile) => {
-		setAddRecordings([...addRecordings, {id: nanoid(), url: blobUrl, src: audiofile}]);
+		setAddRecordings([
+			...addRecordings,
+			{id: nanoid(), url: blobUrl, src: audiofile, value: value},
```

Comment:
Please consider a different property name (e.g. `description`) instead of `value`.